src/views/confirmation/confirmation.njk

```diff
@@ -107,4 +108,14 @@
       }
     ]
   }) }}
+
+  <h2 class="govuk-heading-m">{{ i18n.whatToDoNext }}</h2>
+  <p class="govuk-body">
+    <a href="/tell-companies-house-you-have-verified-someones-identity/confirmation-redirect?id=verify-service-link" class="govuk-link" id="verify-service-link"">{{ i18n.linkToVerifyService }}</a>
```

Comment:
I think you may have an extra closing " here under the id name field: `verify-service-link`